lib/screens/event/event.dart

```diff
@@ -46,55 +45,69 @@ class _EventPageState extends State<EventPage> {
     "gluten": "Gluten"
   };
   void initState() {
-    locator<EventService>()
-        .getEvent(widget.eventId)
-        .then((value) => setState(() {
-              this.event = value;
-            }));
+    locator<EventService>().getEvent(widget.eventId).then((value) => setState(() {
+          this.event = value;
+          if (this.event != null) {
+            this.drinkPackageAnswer = "Alkohol";
```

Comment:
Why do we force drinkPackageAnswer = "Alkohol" on initState?

Reply:
Needs some default and figured thats what most people want when signing up. Other option probably creates more confusion or frustration. Alcohol is opt-out right? 🥴